jest/utils/errorHandling.tsx

```diff
@@ -1,8 +1,8 @@
 const React = require("react");
```

Comment:
Need to validate why the export / import was made that way, and how to adapt the ts config to allow this